bundler/lib/bundler/rubygems_ext.rb

```diff
@@ -282,7 +282,7 @@ def match_platforms?(platform, platforms)
 
   # On universal Rubies, resolve the "universal" arch to the real CPU arch, without changing the extension directory.
   class BasicSpecification
-    if /^universal\.(?<arch>.*?)-/ =~ (CROSS_COMPILING || RUBY_PLATFORM)
+    if /^universal\.(?<arch>.*?)-/ =~ CROSS_COMPILING || RUBY_PLATFORM
```

Comment:
This is a false positive in RuboCop. I've opened PR https://github.com/rubocop/rubocop/pull/12607, but until that is resolved, it'll need to prevent false detections by using `# rubocop:disable Style/RedundantParentheses`.

Reply:
Thanks for your input!